src/core/preload.cc

```diff
@@ -127,22 +120,37 @@ namespace SSC {
       "  Object.freeze(globalThis.__args.config);                            \n"
       "  Object.freeze(globalThis.__args.argv);                              \n"
       "  Object.freeze(globalThis.__args.env);                               \n"
+      "                                                                      \n"
+      "  try {                                                               \n"
+      "    const event = '__runtime_init__';                                 \n"
+      "    let onload = null                                                 \n"
+      "    Object.defineProperty(globalThis, 'onload', {                     \n"
+      "      get: () => onload,                                              \n"
+      "      set (value) {                                                   \n"
+      "        const opts = { once: true };                                  \n"
+      "        if (onload) {                                                 \n"
+      "          globalThis.removeEventListener(event, onload, opts);        \n"
```

Comment:
we probably don't need to remove it as it has `{ once: true }` already, but this is fine